optaweb-employee-rostering-frontend/public/assets/images/extreme.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think these files shouldn't be in this PR, right? Incl. the COVID diagrams etc.